src/components/ContentHighlights/data/reducer.js

```diff
@@ -0,0 +1,27 @@
+/*eslint-disable*/ 
+import { logError } from '@edx/frontend-platform/logging';
+import {
+  SET_HIGHLIGHT_STEPPER_MODAL,
+  SET_CURRENT_STEPPER_STEP,
+} from './actions';
+
+export const initialStepperModalState = {
+  isOpen: false,
+  step: 0,
+  highlight: null,
+  isLoading: false,
```

Comment:
[curious] What will this `isLoading` variable be used for?

Reply:
Co-Tabnine-pilot thought it was useful at the moment. Removed since not in use.